### DISCUSSION TITLE
CityEnergyAnalyst v3.32.0

### DISCUSSION THREAD

**Discussion by reyery** [architecture-building-systems/CityEnergyAnalyst#3264] [Announcements]
In this release, we fix the issue some users are facing, where the GUI does not launch after using the previous windows installation.

From the change log:
- 2022-11-10 - 3.31.0 - #3262 CEA installation fix
- 2022-11-09 - 3.31.0 - #3261 Automate windows setup build
- 2022-10-25 - 3.31.0 - #3258 Create setup_build.yml
- 2022-09-02 - 3.31.0 - #3204 3163 new objective functions for dc
- 2022-08-17 - 3.31.0 - #3222 Gets building height from OSM in zone helper
- 2022-08-16 - 3.31.0 - #3227 Use micromamba instead of miniforge
- 2022-08-10 - 3.31.0 - #3208 fixing overlapping geometries
- 2022-08-08 - 3.31.0 - #3216 Fix CRS change
- 2022-08-05 - 3.31.0 - #3212 Handling potential CRS mismatch
- 2022-08-03 - 3.31.0 - #3209 Fix memory error when writing radiance surfaces
- 2022-08-03 - 3.31.0 - #3185 -remove custom-roof-coverage parameter
- 2022-07-19 - 3.31.0 - #3187 3137 VCC model update
- 2022-07-19 - 3.31.0 - #3196 Update schemas.yml
- 2022-07-12 - 3.31.0 - #3190 Update glossary documentation
- 2022-06-23 - 3.31.0 - #3172 Docker daysim fix
- 2022-06-22 - 3.31.0 - #3188 Fix error when no standards are found
- 2022-06-22 - 3.31.0 - #3181 Refactoring of optimization algorithm - Part II
- 2022-06-21 - 3.31.0 - #3161 3131 capacity correction for vcc & free cooling
- 2022-06-10 - 3.31.0 - #3182 Add os matrix to test workflow
- 2022-06-09 - 3.31.0 - #3184 Fix parsing years with numeric string
- 2022-06-06 - 3.31.0 - #3175 Move daysim binaries
- 2022-06-03 - 3.31.0 - #3160 3159 unit test for district cooling activation
- 2022-05-31 - 3.31.0 - #3174 Allow more than three use types
- 2022-05-28 - 3.31.0 - #3168 Run tests when pushed to master
- 2022-05-13 - 3.31.0 - #3165 Create GitHub Actions to trigger `cea test --workflow quick` for PRs
- 2022-05-12 - 3.31.0 - #3167 Release 3.31.0

<hr /><em>This discussion was created from the release <a href='https://github.com/architecture-building-systems/CityEnergyAnalyst/releases/tag/v3.32.0'>v3.32.0</a>.</em>

**Comment by lguilhermers** [comment]
A couple of users which opened issues related to the interface before said they tried again with this version and the error still persits. I might be able myself tomorrow and will update here.
